releases/release-1.26/release-notes/maps/pr-112058-map.yaml

```
@@ -0,0 +1,3 @@
+pr: 112058
+releasenote:
+  text: Updated `cri-tools` to [v1.25.0(https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.25.0)
```

Missing closing bracket:

```suggestion
  text: Updated `cri-tools` to [v1.25.0](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.25.0)
```